cypress/integration/run-processes/run-import-d2cc-process-acceptance-test.js

```diff
@@ -48,8 +48,8 @@ describe('Check CSE D2CC import corner runs correctly', () => {
         gc.setDate(DATE)
         gc.setTime(TIME)
         gc.statusInTimestampViewShouldBe(gc.READY, TIMEOUT_UPLOAD)
-        gc.runButtonShouldBeEnabled()
-        gc.runComputation()
+        gc.runButtonForBusinessViewShouldBeEnabled(Date.parse(DATE + 'T' + TIME))
```

Comment:
rename method ( this method is not only used in the business view)

Reply:
the method has been renamed